### DISCUSSION TITLE
Angular 13 or above Example that works?

### DISCUSSION THREAD

**Discussion by bobskigit** [aws/aws-iot-device-sdk-js-v2#332] [General]
Is there a working example using MQTT for Angular 13 and above?

I've seen many examples using other IoT brokers but none using AWS

**Comment by jmklix** [comment]
No, we don't have any [samples](https://github.com/aws/aws-iot-device-sdk-js-v2/tree/main/samples) using angular 13 or above. But I welcome anyone else to share their examples below.
